proxmox/ssh/client.go

```diff
@@ -269,8 +270,31 @@ func (c *client) NodeUpload(
 			remoteFilePath, bytesUploaded, fileSize)
 	}
 
+	remoteStat, err := remoteFile.Stat()
```

Comment:
the `NodeUpload` is not used anymore, we can remove this method from the interface and delete this implementation as well.

Reply:
Alright, I'll remove the logic for the file mode again. But I guess removing the whole implementation is out-of-scope for this PR, isn't it? If not I can of course remove it too.

Reply:
yeah, just leave it, we can do the cleanup separately